src/components/Table/Table.jsx

```diff
@@ -52,6 +52,8 @@ const propTypes = {
     hasRowSelection: PropTypes.oneOf(['multi', 'single', false]),
     hasRowExpansion: PropTypes.bool,
     hasRowNesting: PropTypes.bool,
+    /** If the hierarchy only has 1 nested level of children */
+    hasSingleNestedHierarchy: PropTypes.bool,
```

Comment:
This is just an idea, but perhaps a name like "hasSingleLevelRowNesting" would be easier to understand. Or maybe it is even better to expand the existing `hasRowNesting` the same way `hasFilter` has been expanded to be either a boolean or some additional configuration data. This extra config data can then be more specific, e.g. {addOffset: false}. With the later option we don't have to introduce a new option prop just for this and if we in the future want to add more rowNesting configurations we are on the right track.

Reply:
Great idea. I confuse myself with the syntax around nesting, hierarchy, and expanding as well so this would help. I'll expand upon the `hasRowNesting`

Reply:
Went with `hasRowNesting: false` OR `hasRowNesting: { hasSingleLevelRowNesting: true }`